### ISSUE TITLE
 Backspace and Enter Stopped working

### ISSUE THREAD

**Issue by DBruceLane** [yzhang-gh/vscode-markdown#1400]
Type: <b>Bug</b>

I could write any character, but could not use backspace or enter

Keys were working on other applications

Extension version: 3.6.2
VS Code version: Code 1.87.0 (019f4d1419fbc8219a181fab7892ebccf7ee29a2, 2024-02-27T23:41:44.469Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz (8 x 2803)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|31.71GB (21.01GB free)|
|Process Argv|. --crash-reporter-id 925dd1b8-5a57-4f27-a17a-a4e0bbb44da7|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30256859
vscorecescf:30445987
vscod805cf:30301675
binariesv615:30325510
vsaa593:30376534
py29gd2263:30899288
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
welcomedialog:30910333
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
2e7ec940:30968132
pythontbext0:30879054
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
d6fi2104:30959263
d34g3935:30971562
edj9j583:30969295
a89i1917:30973842
dsvsc020:30976470

```

</details>

**Comment by distbit0** [comment]
this is very much affecting me to the extent that I have had to disable this extension for the time being :/

**Comment by distbit0** [comment]
I fixed this by unmapping the backspace key in shortcuts

**Comment by yzhang-gh** [comment]
Hi, sorry for the late reply. You can check this [FAQ](https://github.com/yzhang-gh/vscode-markdown?tab=readme-ov-file#q-the-extension-is-unresponsive-causing-lag-etc-performance-issues) and also [this one](https://github.com/yzhang-gh/vscode-markdown?tab=readme-ov-file#q-error-command-markdownextensiononxxxkey-not-found) where you may find the same "solution" as above

![image](https://github.com/yzhang-gh/vscode-markdown/assets/7588612/cb92d3d1-9161-49d6-a804-8ac149345c35)

In short, it is because there are other extensions conflicting with this one, e.g. Spell Right, Todo Tree...
